logging/CMakeLists.txt

```diff
@@ -8,3 +7,7 @@ target_link_libraries(simple_logger common rc-fshare)
 
 add_executable(log_converter log_converter.cpp)
 target_link_libraries(log_converter common rc-fshare)
+
+# We need XML because Context includes configurables,
+# which uses XML to represent the tree.
+qt5_use_modules(log_converter Xml Core)
```

Comment:
This feels unrelated to this pr. Is this part of the process to save context every frame as a data logging solution or something?

Reply:
No, this is because the per-robot configurables got added into context, which log_converter depends upon. When this isn't here log_converter will (eventually) include context, which will include some Qt files from the Xml package, so it will fail. The "proper" fix is that `qt5_use_modules` should transitively allow the include directories (the macro is kind of broken, but it's not our code) - however, this is a reasonable workaround.